afids_utils/tests/test_afids.py

```diff
@@ -18,7 +18,7 @@
 
 
 @pytest.fixture
-def valid_fcsv_file() -> PathLike[str]:
+def valid_file() -> PathLike[str]:
```

Comment:
Not a new issue, but I think the return type here should just be `Path`, because we know that's what we're producing rather than a `str` or something